server/data/orchestrationApiClient.ts

```diff
@@ -82,6 +83,21 @@ export default class OrchestrationApiClient {
     })
   }
 
+  async getBookedVisitCountByDate(prisonId: string, date: string): Promise<number> {
+    const visits = await this.restClient.get<PageVisitDto>({
+      path: `/visits/search`,
+      query: new URLSearchParams({
+        prisonId,
+        visitStartDate: date,
+        visitEndDate: date,
+        visitStatus: 'BOOKED',
+        page: '0',
+        size: '1',
```

Comment:
Just using visits `/search` endpoint here to get visit count. Only need to return a single result as that still gives what we need; i.e. `totalElements`.